### PR TITLE
Clarify that using printf formatting is for security

### PR DESCRIPTION
**Pull request by Yajo**
See https://github.com/OCA/pylint-odoo/issues/302.

### CODE REVIEW

**Comment by luisg123v** [comment]
FYI @moylop260

**Review by luisg123v — APPROVED** [review]

**Review by moylop260 — APPROVED** [review]
👍 

Should we add a reference to:
 - https://lucumr.pocoo.org/2016/12/29/careful-with-str-format
?

**Comment by moylop260** [comment]
@sbidoul @gurneyalex 

Could you give me permissions to merge in this project, please?

File: website/Contribution/CONTRIBUTING.rst [hunk 1]
```diff
@@ -607,7 +607,7 @@ Idioms
 * For Python 3 (Odoo >= 11.0), no need for utf-8 coding line as this is
   implicit.
 * Prefer `%` over `.format()`, prefer `%(varname)` instead of positional.
-  This is better for translation and clarity.
+  This is better for translation and security.
```
Comment (sbidoul) [inline website/Contribution/CONTRIBUTING.rst:610]: It would be nice to have a link to some source that explains why it is a security issue.

**Comment by sbidoul** [comment]
> Could you give me permissions to merge in this project, please?

Hi @moylop260 currently only board has access to this repo, probably because it is (was?) the OCA website.
We can open the discussion on who should be added, but in the meantime, just @ mention me and I'll review and merge.

**Pull request merged**